docs/how-to/kubernetes.md

```diff
@@ -351,6 +351,47 @@ matches = queried_docs[0].matches
 print(f"Matched documents: {len(matches)}")
 ```
 
+## Update your Executor in Kubernetes
+
+In Kubernetes, you can update your Executors by patching the deployment corresponding to your Executor.
+
+For instance, in the example above, we may want to change the `batch_size` parameter of the CLIPEncoder.
+
+To do this, change the content of the Deployment inside the `executor.yml` dumped by `.to_kubernetes_yaml`.
+
+You need to update the `arguments` passed to the `container` inside the `Deployment` as follows:
+
+```yaml
+    spec:
+      containers:
+      - args:
+        - executor
+        - --name
+        - encoder
+        - --k8s-namespace
+        - custom-namespace
+        - --uses
+        - config.yml
+        - --port
+        - '8080'
+        - --uses-metas
+        - '{}'
+        - --uses-with
+        - '{"batch_size": 64}'
+        - --native
+        command:
+        - jina
+```
+
+After doing so, you can run:
+
+```shell script
+kubectl patch deployment encoder --patch-file ./k8s_flow/encoder/encoder.yml -n k8s-namespace
+```
+
+This will make sure that the Pods are restarted with the new parameters, which will make the Executor to be restarted in a new docker container 
```

Comment:
```suggestion
This will make sure that the Pods are restarted with the new parameters, which will make the Executor to be restarted in a new Docker container 
```